src/containers/Playlists.js

```diff
@@ -5,16 +5,55 @@ import { decode } from 'he'
 import LinkItem from '../components/LinkItem'
 
 class Playlists extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
```

Comment:
Eventually I want to highlight the currently playing playlist in the '/' route. This means that there will need to be a consistant way to set the index of this playlist through search returns and pagination.